src/ducks/Studio/timeseries/hooks.js

```diff
@@ -97,6 +98,8 @@ export function useTimeseries (
       let raceCondition = false
       let mergedData = []
 
+      const hasDaaMetric = metrics.some(({key}) => key === Metric.daily_active_addresses.key)
```

Comment:
Since `metrics` is an array, `.includes` is a better way to check existence of the element inside the array.